features/startup.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Startup
```

Comment:
It's a little bit ambiguous as we're only testing with the `-d` flag and not without it.